### PR TITLE
feat: lib utils cleanup

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

Removes some spotted useless utils:

## Issue (if applicable)

N/A

## Risk

> High Risk PRs Require 2 approvals

Low

> What protocols, transaction types, wallets or contract interactions might be affected by this PR?

## Testing

- proceed to a paranoia test of bps and quote amounts and ensure things still look good amounts-wise: no extra 0s or super low precision

### Engineering

- ^

### Operations

- [ ] :checkered_flag: My feature is behind a flag and doesn't require operations testing (yet)

- ^

### CODE REVIEW

File: packages/utils/src/basisPoints.ts [hunk 1]
```diff
@@ -1,5 +1,7 @@
-import type { BigNumber } from './bignumber/bignumber'
-import { bn, bnOrZero } from './bignumber/bignumber'
+import { bn } from '@shapeshiftoss/chain-adapters'
```
Comment (gomesalexandre) [inline packages/utils/src/basisPoints.ts:1]: no

File: src/lib/bignumber/bignumber.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import { fromBaseUnit } from '@shapeshiftoss/utils'
```
Comment (gomesalexandre) [inline src/lib/bignumber/bignumber.ts:1]: no

Comment (gomesalexandre) [reply src/lib/bignumber/bignumber.ts]: fd0837a99b

File: packages/utils/src/basisPoints.ts (hunk 1 above)
Comment (gomesalexandre) [reply packages/utils/src/basisPoints.ts]: fd0837a99b

**Comment by kaladinlight** [comment]
remove from `package/utils/src/bignumber` as well? or just waiting on cleaning up the dupe bignumber utils?

**Comment by gomesalexandre** [comment]
> remove from `package/utils/src/bignumber` as well? or just waiting on cleaning up the dupe bignumber utils?

Might as well - not sure yet how much effort it will be to get web packages to work in `packages` so that may or may not be a now thing! daf2518564

**Review by kaladinlight — APPROVED** [review]

**Pull request merged**